src/renderers/label.util.ts

```diff
@@ -34,3 +37,23 @@ export const getElementLabelObject = (withLabel: ControlElement): ILabelObject =
         return new LabelObject(deriveLabel(withLabel), true);
     }
 };
+const isRequired = (schema: JsonSchema, schemaPath: string): boolean => {
+    const pathSegments = schemaPath.split('/');
+    const lastSegment = pathSegments[pathSegments.length - 1];
+    const nextHigherSchemaSegments = pathSegments.slice(0, pathSegments.length - 2);
+    const nextHigherSchemaPath = nextHigherSchemaSegments.join('/');
+    const nextHigherSchema = resolveSchema(schema, nextHigherSchemaPath);
+    if (nextHigherSchema.required !== undefined &&
```

Comment:
What if the path could not be resolved, i.e. `nextHigherSchema` is undefined? I suppose that might happen.